src/ReefGuideAPI.jl

```diff
@@ -122,13 +142,26 @@ function setup_regional_data(config::Dict)
         )
     end
 
+    regional_assessment_data["reef_outlines"] = ""
+    regional_assessment_data["region_long_names"] = ""
+
     # Store cache on disk to avoid excessive cold startup times
     @debug "Saving regional data cache to disk"
     serialize(reg_cache_fn, regional_assessment_data)
 
     # Remember, `@eval` runs in global scope.
     @eval const REGIONAL_DATA = $(regional_assessment_data)
 
+    reef_outline_path = joinpath(reef_data_path, "rrap_canonical_outlines.gpkg")
+    REGIONAL_DATA["reef_outlines"] = GDF.read(reef_outline_path)
+
+    REGIONAL_DATA["region_long_names"] = Dict(
+        "FarNorthern" => "Far Northern Management Area",
+        "Cairns-Cooktown" => "Cairns/Cooktown Management Area",
+        "Townsville-Whitsunday" => "Townsville/Whitsunday Management Area",
+        "Mackay-Capricorn" => "Mackay/Capricorn Management Area"
+    )
+
```

Comment:
Same comment as above - duplicate code?

Could move into a private function?

Reply:
You could, though arguably it is a data concern. I suppose ideally we would have a file in the input data that provides this mapping. So if you imagine heading a 5th region, we wouldn't need to change any code. But that's more effort.